### DISCUSSION TITLE
make 2 images into one

### DISCUSSION THREAD

**Discussion by alen978** [lllyasviel/Fooocus#3393] [Q&A] [answered]
Is it possible to make image 1 and image 2 into image 3
For image 3, keep image 1 original, make image 1 at the top of the image2, any feature can realize it? tried several times, failed.
![merge](https://github.com/user-attachments/assets/23abaaf5-5c23-4fc5-85ca-ed639e7e5eaf)

**Comment by mashb1t** [comment, marked as answer]
You can follow these steps:
1. use Paint / GIMP / Photoshop to add image 1 on top of image 2 with desired position / size
2. use Fooocus to inpaint around the edges of image 1 to blend it in, also adding shadows or structure at the edges
